Automattic/wp-calypso
# My Home: Remove unused cards

#### Changes proposed in this Pull Request

While working on https://github.com/Automattic/wp-calypso/pull/58910, I noted that Calypso maintained cards for My Home that are no longer needed after [the refactor done by the My Home pod a few months ago](https://github.com/Automattic/wp-calypso/pulls?q=is%3Apr+is%3Aclosed+label%3A%22My+Home+Pod%22).

This PR cleans up all the unused My Home's cards.

#### Testing instructions

Use the Calypso live link below, and smoke test the My Home page on a variety of scenarios (site just created, site before launch, site after launch). Make sure there are no differences when compared to production (except for the fact that the primary tasks after launch are shuffled on every page load).
